server/models/user.model.js

```diff
@@ -5,6 +5,7 @@ const { Schema } = mongoose;
 const UserSchema = new Schema(
   {
     accessToken: { type: String },
+    activatedAt: { type: Date },
```

Comment:
why do we need such field?

Reply:
On a demo, Łukasz said that we should save the date of the account activation.
In almost all cases it will be of course almost the same as the createdAt date which is automatically saved on storing a user in DB.